.circleci/config.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           name: "Validate HTML"
           command: |
             yarn build
-            html5validator --root build/ --show-warnings
+            html5validator --root build/
```

Comment:
Hi, @Gianfranco97 
Why not?

Reply:
This parameter generates an error with the build, The problem is that this error is not easily solvable.

The problem is born in:
``` html
<script type="text/javascript" src="...">
```

The only thing that occurs to me is to create a postscript for the build